### PR TITLE
Removed usages of graphdb.Direction from Cypher compiler

### PR DESCRIPTION
**Pull request by systay**

### CODE REVIEW

**Comment by systay** [comment]
Silly CI choked. Re-running

File: community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala [hunk 1]
```diff
@@ -23,6 +23,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.expressions.{NestedPipeE
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.{ArgumentPipe, ExpandAllPipe, LazyTypes}
 import org.neo4j.cypher.internal.compiler.v2_3.planDescription.InternalPlanDescription.Arguments.{EstimatedRows, LegacyExpression}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.BeLikeMatcher._
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.{ExecutionEngineFunSuite, NewPlannerTestSupport}
 import org.neo4j.graphdb.{Direction, Relationship}
```
Comment (pontusmelke) [inline community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionAcceptanceTest.scala]: Shouldn't `Direction` be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/mutation/MergePatternActionNullPropertyHandlingTest.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.ExecutionContext
 import org.neo4j.cypher.internal.compiler.v2_3.commands.expressions.Literal
 import org.neo4j.cypher.internal.compiler.v2_3.commands.{RelatedTo, SingleNode, VarLengthRelatedTo}
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.QueryStateHelper
-import org.neo4j.cypher.internal.frontend.v2_3.InvalidSemanticsException
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, InvalidSemanticsException}
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/mutation/MergePatternActionNullPropertyHandlingTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/parser/CypherParserTest.scala [hunk 3]
```diff
@@ -31,7 +31,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.values.TokenType.Propert
 import org.neo4j.cypher.internal.compiler.v2_3.helpers.LabelSupport
 import org.neo4j.cypher.internal.compiler.v2_3.mutation._
 import org.neo4j.cypher.internal.compiler.v2_3.ast.convert.commands.StatementConverters._
-import org.neo4j.cypher.internal.frontend.v2_3.SyntaxException
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, SyntaxException}
 import org.neo4j.cypher.internal.frontend.v2_3.parser._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/parser/CypherParserTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/parser/PatternPartTest.scala [hunk 4]
```diff
@@ -22,7 +22,7 @@ package org.neo4j.cypher.internal.compiler.v2_3.parser
 import org.neo4j.cypher.internal.compiler.v2_3.ast.convert.commands.PatternConverters._
 import org.neo4j.cypher.internal.compiler.v2_3.commands.expressions.Literal
 import org.neo4j.cypher.internal.compiler.v2_3.commands.{Pattern => LegacyPattern, _}
-import org.neo4j.cypher.internal.frontend.v2_3.ast
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, ast}
 import org.neo4j.cypher.internal.frontend.v2_3.parser.{Expressions, ParserTest, Patterns}
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/parser/PatternPartTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/ExpandAllPipeTest.scala [hunk 5]
```diff
@@ -27,6 +27,7 @@ import org.mockito.stubbing.Answer
 import org.neo4j.cypher.internal.compiler.v2_3.ExecutionContext
 import org.neo4j.cypher.internal.compiler.v2_3.spi.QueryContext
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.{Direction, Node, Relationship}
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/ExpandAllPipeTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/OptionalExpandAllPipeTest.scala [hunk 6]
```diff
@@ -27,6 +27,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.ExecutionContext
 import org.neo4j.cypher.internal.compiler.v2_3.commands.predicates.{Not, Predicate, True}
 import org.neo4j.cypher.internal.compiler.v2_3.spi.QueryContext
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.{Direction, Node, Relationship}
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/OptionalExpandAllPipeTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/OptionalExpandIntoPipeTest.scala [hunk 7]
```diff
@@ -27,6 +27,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.ExecutionContext
 import org.neo4j.cypher.internal.compiler.v2_3.commands.predicates.{Not, Predicate, True}
 import org.neo4j.cypher.internal.compiler.v2_3.spi.QueryContext
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.{Direction, Node, Relationship}
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/OptionalExpandIntoPipeTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/HistoryTest.scala [hunk 8]
```diff
@@ -20,6 +20,7 @@
 package org.neo4j.cypher.internal.compiler.v2_3.pipes.matching
 
 import org.neo4j.cypher.internal.compiler.v2_3._
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.{Direction, DynamicRelationshipType, Relationship}
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/HistoryTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/PatternRelationshipTest.scala [hunk 9]
```diff
@@ -20,6 +20,7 @@
 package org.neo4j.cypher.internal.compiler.v2_3.pipes.matching
 
 import org.neo4j.cypher.internal.compiler.v2_3.commands.expressions.Literal
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/PatternRelationshipTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/StepSizeTest.scala [hunk 10]
```diff
@@ -20,42 +20,43 @@
 package org.neo4j.cypher.internal.compiler.v2_3.pipes.matching
 
 import org.neo4j.cypher.internal.compiler.v2_3.commands.predicates.True
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/matching/StepSizeTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planDescription/PlanDescriptionArgumentSerializerTests.scala [hunk 11]
```diff
@@ -21,6 +21,7 @@ package org.neo4j.cypher.internal.compiler.v2_3.planDescription
 
 import org.neo4j.cypher.internal.compiler.v2_3.planDescription.InternalPlanDescription.Arguments.{ExpandExpression,
 EstimatedRows, DbHits, Rows}
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planDescription/PlanDescriptionArgumentSerializerTests.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planDescription/RenderTreeTableTest.scala [hunk 12]
```diff
@@ -27,6 +27,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.predicates.{Equals, HasL
 import org.neo4j.cypher.internal.compiler.v2_3.commands.values.{TokenType, KeyToken}
 import org.neo4j.cypher.internal.compiler.v2_3.pipes._
 import org.neo4j.cypher.internal.compiler.v2_3.planDescription.InternalPlanDescription.Arguments._
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planDescription/RenderTreeTableTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/QueryGraphTest.scala [hunk 13]
```diff
@@ -20,6 +20,7 @@
 package org.neo4j.cypher.internal.compiler.v2_3.planner
 
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans.{IdName, PatternRelationship, SimplePatternLength}
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/QueryGraphTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/NodeHashJoinPlanningIntegrationTest.scala [hunk 14]
```diff
@@ -25,7 +25,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans._
 import org.neo4j.cypher.internal.compiler.v2_3.planner.{LogicalPlanningTestSupport2, PlannerQuery}
 import org.neo4j.cypher.internal.frontend.v2_3.ast._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
-import org.neo4j.cypher.internal.frontend.v2_3.{LabelId, PropertyKeyId}
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, LabelId, PropertyKeyId}
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/NodeHashJoinPlanningIntegrationTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/cardinality/triplet/TripletConverterTest.scala [hunk 15]
```diff
@@ -26,7 +26,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.cardinality.{Spec
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans.{IdName, PatternRelationship, SimplePatternLength}
 import org.neo4j.cypher.internal.frontend.v2_3.ast._
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
-import org.neo4j.cypher.internal.frontend.v2_3.{LabelId, RelTypeId, SemanticTable}
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, LabelId, RelTypeId, SemanticTable}
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/cardinality/triplet/TripletConverterTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/greedy/FindShortestPathsTest.scala [hunk 16]
```diff
@@ -21,6 +21,7 @@ package org.neo4j.cypher.internal.compiler.v2_3.planner.logical.greedy
 
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans.{PatternRelationship, ShortestPathPattern, _}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.{LogicalPlanningTestSupport2, QueryGraph}
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/greedy/FindShortestPathsTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/idp/ExpandSolverStepTest.scala [hunk 17]
```diff
@@ -26,7 +26,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.steps.LogicalPlan
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.{Cardinality, LogicalPlanningContext, Metrics, QueryGraphSolver}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.{CardinalityEstimation, LogicalPlanConstructionTestSupport, PlannerQuery, QueryGraph}
 import org.neo4j.cypher.internal.compiler.v2_3.spi.PlanContext
-import org.neo4j.cypher.internal.frontend.v2_3.SemanticTable
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, SemanticTable}
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/idp/ExpandSolverStepTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/rewriter/UnnestEmptyApplyTest.scala [hunk 18]
```diff
@@ -22,6 +22,7 @@ package org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans.rewriter
 import org.neo4j.cypher.internal.compiler.v2_3.helpers.Converge.iterateUntilConverged
 import org.neo4j.cypher.internal.compiler.v2_3.planner.LogicalPlanningTestSupport
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans._
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/rewriter/UnnestEmptyApplyTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/rewriter/UnnestOptionalTest.scala [hunk 19]
```diff
@@ -22,7 +22,7 @@ package org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans.rewriter
 import org.neo4j.cypher.internal.frontend.v2_3.ast.{SignedDecimalIntegerLiteral, PropertyKeyName, Property, Equals}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.{PlannerQuery, LogicalPlanningTestSupport}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans._
-import org.neo4j.cypher.internal.frontend.v2_3.LabelId
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, LabelId}
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/plans/rewriter/UnnestOptionalTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/steps/ExtractBestPlanTest.scala [hunk 20]
```diff
@@ -28,7 +28,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.spi.PlanContext
 import org.neo4j.cypher.internal.frontend.v2_3.ast._
 import org.neo4j.cypher.internal.frontend.v2_3.notification.{IndexHintUnfulfillableNotification, JoinHintUnfulfillableNotification}
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
-import org.neo4j.cypher.internal.frontend.v2_3.{IndexHintException, JoinHintException}
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, IndexHintException, JoinHintException}
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/steps/ExtractBestPlanTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/steps/SolveOptionalMatchesTest.scala [hunk 21]
```diff
@@ -25,7 +25,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.Metrics.QueryGrap
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.plans._
 import org.neo4j.cypher.internal.compiler.v2_3.planner.logical.{Cardinality, LogicalPlanningContext, QueryPlannerConfiguration}
 import org.neo4j.cypher.internal.compiler.v2_3.planner.{CardinalityEstimation, LogicalPlanningTestSupport, PlannerQuery, QueryGraph}
-import org.neo4j.cypher.internal.frontend.v2_3.{SemanticTable, ast}
+import org.neo4j.cypher.internal.frontend.v2_3.{SemanticDirection, SemanticTable, ast}
 import org.neo4j.cypher.internal.frontend.v2_3.test_helpers.CypherFunSuite
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/logical/steps/SolveOptionalMatchesTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/PatternMatchingTest.scala [hunk 22]
```diff
@@ -27,12 +27,13 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.values.UnresolvedLabel
 import org.neo4j.cypher.internal.compiler.v2_3.executionplan.builders.PatternGraphBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.matching.PatternMatchingBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/PatternMatchingTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/PatternNodeTest.scala [hunk 23]
```diff
@@ -21,6 +21,7 @@ package org.neo4j.cypher.internal.compiler.v2_3
 
 import org.neo4j.cypher.GraphDatabaseFunSuite
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.matching.{MatchingPair, PatternNode}
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/PatternNodeTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/ScalaPatternMatchingTest.scala [hunk 24]
```diff
@@ -25,12 +25,13 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.expressions.Literal
 import org.neo4j.cypher.internal.compiler.v2_3.executionplan.builders.PatternGraphBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.matching.PatternMatchingBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/ScalaPatternMatchingTest.scala]: Shouldn't Direction be removed here as well?

File: community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/SimplePatternMatchingTest.scala [hunk 25]
```diff
@@ -24,6 +24,7 @@ import org.neo4j.cypher.internal.compiler.v2_3.commands.RelatedTo
 import org.neo4j.cypher.internal.compiler.v2_3.executionplan.builders.PatternGraphBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.pipes.matching.SimplePatternMatcherBuilder
 import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.SemanticDirection
 import org.neo4j.cypher.internal.frontend.v2_3.symbols._
 import org.neo4j.graphdb.Direction
```
Comment (pontusmelke) [inline community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/SimplePatternMatchingTest.scala]: Shouldn't Direction be removed here as well?

**Pull request merged**